app/event/send-batch-processing-events.js

```diff
@@ -1,14 +1,25 @@
+const { v4: uuidv4 } = require('uuid')
 const raiseEventBatch = require('./raise-event-batch')
 
-const sendBatchProcessedEvents = async (filename, paymentRequests, sequence, batchExportDate) => {
-  const events = paymentRequests.map(paymentRequest => ({
-    id: paymentRequest.correlationId,
-    name: 'batch-processing',
-    type: 'info',
-    message: 'Payment request created from batch file',
-    data: { filename, sequence, paymentRequest, batchExportDate }
-  }))
-  await raiseEventBatch(events)
+const sendBatchProcessedEvents = async (paymentRequests, filename, sequence, batchExportDate) => {
+  if (paymentRequests?.length) {
+    try {
+      const events = paymentRequests?.map(paymentRequest => ({
+        id: uuidv4(),
+        name: 'batch-processing',
+        type: 'info',
+        message: 'Payment request created from batch file',
+        data: {
+          filename,
+          sequence,
+          batchExportDate,
+          paymentRequest
+        }
+      }))
+
+      await raiseEventBatch(events)
+    } catch {}
```

Comment:
Should we at least log something so it's visible in App Insights if there's an error thrown and we've failed to produce the events?

Also what's the rationale for allowing the error to still continue processing?  Is there an impact on the MI report if we allow the requests to go through but we're not publishing the first event?